src/controller/index.ts

```diff
@@ -67,3 +94,43 @@ export const newAutomatedLetter = async (req: Request<{}, {}, NewsPost>, res: Re
         });
     }
 };
+
+export const callBackTwitter = async (req: Request<{}, {}>, res: Response) => {
+    try {
+        const oauth_token = req.query.oauth_token as string;
+        const oauth_verifier = req.query.oauth_verifier as string;
+        const { oauth_token: accessToken, oauth_token_secret: accessSecret } = await twitterClient.getAccessToken({
+            oauth_verifier,
+            oauth_token,
+        });
+        res.status(200).json({
+            accessToken,
+            accessSecret,
+        });
+    } catch (error) {
+        console.error(error);
+        res.status(500).send("An error occurred.");
+    }
+};
+
+export const authorizeTwitter = async (req: Request<{}, {}>, res: Response) => {
+    try {
+        let oauth_token = "",
```

Comment:
`oauth_token` should be renamed to `oAuthToken`
`oauth_token_secret` should be renamed to `oAuthTokenSecret`

use camelCasing for all variables, if a 3rd party API uses another naming convention, rename where needed.

`authURL.searchParams.append("oauth_token", oauth_token)`
should be rewritten as 
`authURL.searchParams.append("oauth_token", oAuthToken)`